README.md

```diff
@@ -133,6 +139,36 @@ results collect: ['PATH/bitrate_buffer.720p_files/encapp_XXX.json'
 ```
 
 
+## 3.3 Video decoding/encoding in series
+
+Instead of using yuv video data and encoded video can be used as a source.
+A suitable decoder will be chosen by the system (normally hardware decoders are prioritized).
+```
+$ encapp.py run tests/bitrate_transcoder_show.pbtxt
+...
+results collect: ['PATH/bitrate_transcoder_show_files/encapp_XXX.json']
+```
+
+This example also show a different feature: visualization. It is possible to show videos in a grid like pattern (layout is automatic). 
+
+The parameter for showing the video is 'show' in the 'input' section. Currently there is no provision for visualizing the encoded result (which would require an additional decoder). Likewise the encoder is not available for explicit setting at the moment.
+
+
+
+## 3.4 Video encoding using camera source
+
+Encapp also supports camera using the Camera2 API.
+```
+$ encapp.py run tests/camera_parallel.pbtxt 
+...
+['testing_Cipher_Placeholder_2022-08-24_14_05/camera_parallel_files/encapp_6b406f7d-b28f-4b64-b2c4-94d8ab41487e.json', 'PATH/camera_parallel_files/encapp_XXX.json']
+```
+
+This example will run two encodings with the camera as source in parallell and show the viewfinder in fullscreen. 
```

Comment:
parallel (typo)

Reply:
Ooop. Had a editor malfunction so I missed this edit.